### DISCUSSION TITLE
Comment generer 2 dates à utiliser comme Durables ?

### DISCUSSION THREAD

**Discussion by jhuon1** [postmanlabs/postman-flows#325] [Q&A]
Bonjour à tous,

J'essaye de convertir quelques Collections Run que nous avons déjà avec cette cool feature Flows... Mais dès que je commence, je bloque. Je n'arrive pas à trouver un moyen de créer une variable simple comme une date, genre startDate et EndDate en fonction de la date d'aujourd'hui...

C'est super simple dans les scripts classic des Run, mais là, je sèche...
J'ai bien compris qu'on ne veut pas de script block et pas de JS dans Flows... Cependant quelque fonctions de dates ou mathématique de base sont essentielles pour des mini transformations de données.

Avez-vous un moyen de contourner celà en attendant plus de blocks de ce genre ?
Je pensais dériver la sortie Test de send request pour récupérer une valeur dynamique que je passerai dans le "name" de la structure de reponse test... ce qui je vous l'accorde est vraiment crade...

data:
  - index: 0
    name: **réponse 200 sur le GET /xxx**
    passed: true
    skipped: false
    error: null

Sinon, j'ai bien peur que Flows ne soit pas encore pour nous, et je serai surpris que d'autres ne soient pas dans ce même cas de figure.

D'avance merci pour vos idées et merci aux devs sur ce projet pour toutes les réponses de ce forum.

**Comment by chrisjpatty** [comment]
Hey @jhuon1! We're actively working on expanding the primitive blocks as we speak actually, and Date and Time blocks are coming out very soon. We're also expanding the data manipulation capabilities with more operation blocks. We'd love to hear your feedback on these new blocks when they're released. Are there any operations you're looking for specifically that are missing?
